vdsm_hooks/log_console/before_vm_start.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python3
+#
+# Copyright 2022 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#
+# Refer to the README and COPYING files for full details of the license
+#
+
+from __future__ import absolute_import
+
+import os
+import traceback
+
+from vdsm.hook import hooking
+
+
+def main():
+    enable = os.environ.get('log_console')
+    if enable == 'on':
+        domxml = hooking.read_domxml()
+
+        vm_name = domxml.getElementsByTagName('name')[0].firstChild.data
+
+        consoles = domxml.getElementsByTagName('console')
```

Comment:
Adding this to the 'console' does not work. Adding to 'serial' works, so I will. dumpxml then shows it added to both console and serial.